src/runtime/ObjectTemplate.cpp

```diff
@@ -286,6 +287,66 @@ Object* ObjectTemplate::instantiate(Context* ctx)
     return result;
 }
 
+bool ObjectTemplate::installTo(Context* ctx, Object* target)
+{
+    struct Sender {
+        ObjectTemplate* self;
+        Context* ctx;
+        Object* target;
+    } sender;
+
+    sender.self = this;
+    sender.ctx = ctx;
+    sender.target = target;
+
+    SandBox sb(ctx);
+    auto result = sb.run([](ExecutionState& state, void* data) -> Value {
+        Sender* sender = (Sender*)data;
+        size_t propertyCount = sender->self->m_properties.size();
+        auto properties = sender->self->m_properties;
+        for (size_t i = 0; i < propertyCount; i++) {
+            auto propertyIndex = i;
```

Comment:
Why don't you just use ```i``` instead of ```propertyIndex```? this is a minor opinion of mine

Reply:
I can use just `i`